GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -968,11 +1078,34 @@ private String sanitize(String s) {
                                                 .replaceAll("\n", "")
                                                 .trim();
                             }
+
                             createCell(
                                     row,
                                     offset++,
                                     header,
                                     headerStyle);
+
+                            // check if we need to create columns for all options
+                            if (QuestionType.OPTION == q.getType()  && useQuestionId) {
```

Comment:
Can you cross check but maybe we have some extra conditions that are used in the `useQID` that you may be missing by only using `useQuestionId`?